pkg/upload/js.go

```diff
@@ -210,8 +224,21 @@ func ProcessJs(
 			return fmt.Errorf("could not find a source map, please specify the path by using --source-map")
 		}
 
+		// Ensure that exactly one of --bundle-url and --base-url is specified
+		if bundleUrl == "" && baseUrl == "" {
```

Comment:
We should also validate that if the user provides a directory as `path` then the `baseUrl` is given; and conversely if a single file is given then `bundleUrl` is given. Perhaps this supersedes this check?